knowledge_doc_converter/knowledge_doc_converter/tasks/conversion_task.py

```
@@ -0,0 +1,327 @@
+                )
+            # All retries exhausted - notify backend
+            record_lock_exhausted()
+            record_conversion_skipped("lock_retry_exhausted")
+            try:
+                callback_client.notify_failed(
+                    path=callback_status_path,
+                    document_id=document_id,
+                    generation=index_generation,
+                    error_message="Lock retry exhausted",
+                )
+            except Exception as callback_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)